client/state/terms/schema.js

```diff
@@ -16,7 +16,8 @@ export const termsSchema = {
 								slug: { type: 'string' },
 								description: { type: 'string' },
 								post_count: { type: 'number' },
-								parent: { type: 'number' }
+								parent: { type: 'number' },
+								meta: { type: 'object' }
 							},
 							additionalProperties: false
```

Comment:
> Hmm, I wouldn't expect warnings for properties that aren't part of the schema
This one is one level too deep. We only need `additionalProperties: false` for the patternProperties matches